setup/main.go

```diff
@@ -0,0 +1,250 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"flag"
```

Comment:
In the main codebase we use https://github.com/ogier/pflag (aliased to `flag`) - the main difference is that the Go standard library uses a non-standard way of passing arguments (in particular, `-longname` is permitted, instead of only allowing `--longname`)

Probably good to use that here as well for consistency?

Reply:
Oh whoops--I'm pretty sure I was using pflag here initially, but I think I broke this in refactoring when Go imports helpfully imported the default flag package instead. Good catch; will fix.